src/Apps/Order/Components/Helper.tsx

```diff
@@ -21,8 +21,8 @@ export class Helper extends Component<HelperProps> {
     type: "button",
     flow: "buy now",
   }))
-  onClickReadFAQ(mediator) {
-    mediator.trigger("openOrdersBuyerFAQModal")
```

Comment:
This line is necessary -- please do _not_ merge without adding this back in

Reply:
Woops, got confused with some other recent code I wrote 😄

Reply:
oh... sorry... Was this for analytics tracking?

Reply:
It's okay as the decorator is still there!

Reply:
I think it was merged before I removed the comment.

This will need reaction bump in Force to take effect anyways. @williardx - mind adding it back if this removal broke something plz.

Reply:
@oxaudo We trigger analytics tracking using events in a few places, but this one has the `@track` decorator so we're good!